README.md

```diff
@@ -67,9 +67,15 @@ To hide pagination
 {{< embed-pdf url="./path/to/pdf/file/example.pdf" hidePaginator="true" >}}
 ```
 
+To hide loading spinner
+```
+{{< embed-pdf url="./path/to/pdf/file/example.pdf" hideLoader="true" >}}
+```
+
 ### Parameters
 - **url (required)** : The relative location of the file.  
 - **hidePaginator (optional)**: Boolean which expects `true` or `false`. Hides the paginator for single page documents. 
+- **hideLoader (optional)**: Boolean which expects `true` or `false`. Hides the loading spinner while your document loads. 
```

Comment:
same as above, if you make this `showLoader` the language here would need to change.